templates/actors/parts/actor-spellbook.hbs

```diff
@@ -4,8 +4,7 @@
         <label>{{localize "DND5E.Spellcasting"}}</label>
         {{else}}
         <label>{{localize "DND5E.Level"}}</label>
-        {{numberInput system.details.spellLevel name="system.details.spellLevel" class="spellcasting-level"
-            placeholder="0"}}
+        <input type="text" name="system.details.spellLevel" value="{{system.details.spellLevel}}" class="spellcasting-level" placeholder="0" data-dtype="Number">
```

Comment:
Not sure I agree with this one. Is it really the case that we need to apply deltas to an NPC spellcaster's spellcasting level? I would expect you only ever set it once to a particular value and then leave it alone.

Reply:
We should use the most appropriate input field for the value it is holding. In many cases a `number` input (and our `numberInput` helper) has the advantage that it can catch invalid input early before the form is even submitted. So we should be using `numberInput` with appropriate `min`, `max`, and `step`, for all strictly numeric values except in cases where we want to relax these constraints to allow for deltas. My current intuition is that we only need deltas for HP, XP, and currency. I don't think there are really many other places where the numbers are large enough that deltas are particularly useful.

Reply:
You make a good point here.

Reply:
Updated to resolve this.